terraform/eks_fargate_setup/oidc_thumbprint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [[ -z "${ATLAS_WORKSPACE_SLUG}" ]]; then
```

Comment:
What is `ATLAS_WORKSPACE_SLUG`? I don't see it used anywhere else.

Reply:
This is some cargo cult programming on my end. I can remove.